src/webtransport.js

```diff
@@ -191,16 +258,24 @@ class Http3WTStream {
     }
   }
 
+  /**
+   * @param {StreamWriteEvent} args
+   */
   onStreamWrite(args) {
     // we ignore success
     if (this.pendingoperation) {
       const res = this.pendingres
       this.pendingoperation = null
       this.pendingres = null
-      res()
```

Comment:
Should error, because it indicates a bug in the implementation

Reply:
Or no `this.pendingoperation` guards this already, does `tsc` require an explicit type checking?